src/steps/render.js

```diff
@@ -34,6 +35,13 @@ function createElement(name, ...attrs) {
   return h(name, properties);
 }
 
+function sanitizeJsonLd(jsonLd) {
+  if (jsonLd.toLowerCase().indexOf('</script>') >= 0) {
```

Comment:
`JSON.parse()` would throw anyway if string contained tags

Reply:
no, it could also be quoted:

```html
<html>
<head>
	<script type="application/json-ld">
	{ "$foo": "</script><script type='application/javascript'>alert('hello')</script>" }
	</script>	
</head>
<body>
	test
</body>
</html>
```